encoding.bs

```diff
@@ -1594,13 +1595,18 @@ byteReadable
 
 <ol>
  <li><p>Let <var>bufferSource</var> be the result of
- <a lt="converted to an IDL value">converting</a> <var>chunk</var> to a {{BufferSource}}. If this
- throws an exception, then return a promise rejected with that exception.
-
- <li><p><a>Push</a> a <a lt="get a copy of the buffer source">copy of</a> <var>bufferSource</var> to
- <var>dec</var>'s <a for=TextDecoderStream>stream</a>. If this throws an exception, then return a
+ <a lt="converted to an IDL value">converting</a> <var>chunk</var> to a
```

Comment:
Maybe it's just me, but I feel this "a" should be "an", ie. reading the text as "an allowshared buffersource".